tests/system/python/packages/test_multiple_assets.py

```diff
@@ -339,17 +353,19 @@ def test_multiple_assets_with_reconfig(self, reset_fledge, start_north, read_dat
         # In reconfig number of assets are doubled
         total_assets = total_assets * 2
 
-        # Wait until total_assets are created
-        time.sleep(num_assets + 2 * wait_time)
         verify_ping(fledge_url, skip_verify_north_interface, wait_time, retries)
-        verify_asset(fledge_url, total_assets)
-
-        verify_asset_tracking_details(fledge_url, total_assets, total_benchmark_services, num_assets)
+        
+        # FIX ME: FOGL-7567, Due to increase in timing of asset creation, time spent in reconfig asset also increases.
+        # Remove the WAIT_FIX, Once FOGL-7567 is resolved.
+        WAIT_FIX =  wait_fix + num_assets//100
+        
+        verify_asset(fledge_url, total_assets, WAIT_FIX, wait_time)
```

Comment:
WAIT_FIX is mapped to count? Let's keep as  you wanted. We can remove this with the FOGL-7567 PR itself.